app/dashboard/components/contentRecent.tsx

```diff
@@ -0,0 +1,42 @@
+import Slider from './slider';
+import FilterProjects from './projectFilter';
+import ProjectFetcher from './projectFetcher';
+import styles from '../page.module.scss';
+
+export default function ContentRecent() {
+    const fetchReturn = ProjectFetcher({ userId: 1, fetchMostRecent: false});
+    const { filteredProjects, filter, handleFilterChange } = FilterProjects({projects: fetchReturn.projects});
+
+    return (
+        <div className={styles["recent-projects-container"]}>
```

Comment:
Je hebt 4 nested div voor niets, waarom is dat? Volgens mij kan je dit met 1 div ook doen.